kernel/kernel-api/src/main/java/io/delta/kernel/internal/snapshot/SnapshotManager.java

```diff
@@ -240,6 +240,8 @@ private SnapshotImpl createSnapshot(
    */
   @VisibleForTesting
   public LogSegment getLogSegmentForVersion(Engine engine, Optional<Long> versionToLoadOpt) {
+    final boolean USE_COMPACTED_FILES = true; // TODO: Where to config this
```

Comment:
Does delta-spark have a knob for this? I _think_ that we could have some heuristic for when we should or should not use these compaction files? (Not sure).